### PR TITLE
EUI-3674-mandatory-organisation

### PR DESCRIPTION
**Pull request by rayliangatos**
**Before creating a pull request make sure that:**

- [x] commit messages are meaningful and follow good commit message guidelines
- [x] README and other documentation has been updated / added (if needed)
- [x] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/EUI-3674

### Change description ###
When organisation is not selected and if it's set MANDATORY in CCD config, the validation should be failed.

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

**Review by yarpo — APPROVED** [review]

File: RELEASE-NOTES.md [hunk 1]
```diff
@@ -1,5 +1,7 @@
 ## RELEASE NOTES
-### Version 2.79.2
+### Version 2.79.3
```
Comment (paul-graham) [inline RELEASE-NOTES.md:2]: The version here doesn't match the one in `package.json`.

**Review by paul-graham — CHANGES_REQUESTED** [review]
`package.json` and `RELEASE-NOTES.md` don't agree on the version number.

File: RELEASE-NOTES.md (hunk 1 above)
Comment (rayliangatos) [reply RELEASE-NOTES.md]: the version is matched now

**Review by paul-graham — APPROVED** [review]
Versions now match. :)

**Review by Mo-Lala84 — APPROVED** [review]
approved

**Pull request merged**